### DISCUSSION TITLE
Compatibility of neuralcoref library with Spacy v.3

### DISCUSSION THREAD

**Discussion by khushi810** [explosion/spaCy#10363] [Help: Coding & Implementations] [answered]
There are some issues that have been faced with neuralcoref library and spacy v3.
May I know any updates on this?

Thanks.

**Comment by polm** [comment, marked as answer]
This is already covered in the [sticky issue on neuralcoref](https://github.com/huggingface/neuralcoref/issues/295), but we will not be updating neuralcoref for spaCy v3, and instead are developing a new component. We'll announce it when it's ready.
